source/jormungandr/jormungandr/interfaces/v1/ResourceUri.py

```diff
@@ -65,8 +65,10 @@ def __init__(self, authentication=True, links=True, *args, **kwargs):
             self.get_decorators.append(authentication_required)
 
     def get_filter(self, items, args):
+        if not (items or args.get('tags[]', [])):
+            return ""
```

Comment:
In that case we should return `args["filter"]` unchanged (and it's what would be done without this early-return).
So I think early-return here is not a good idea, as we never know what params will be managed and added below.

That way you should be able to use it safely (and remove those 2 lines from Uri.py shouldn't break tests).